packages/protocol/contracts/governance/Governance.sol

```diff
@@ -657,17 +731,36 @@ contract Governance is
     uint256[] calldata dataLengths,
     bytes32 salt
   ) external {
-    bytes32 hash = keccak256(abi.encode(values, destinations, data, dataLengths, salt));
+    if (isL2()) {
+      bytes32 hash = keccak256(abi.encode(values, destinations, data, dataLengths, salt));
```

Comment:
Can we use getHotfixHash function ?

Reply:
It just ends up costing more gas if I use the function.